### ISSUE TITLE
No running containers

### ISSUE THREAD

**Issue by 1it** [kisev/django-project-deploy#1]
```
$ ./deploy.sh local -u root
Vault password:

PLAY [env-hosts] ***************************************************************

TASK [Install Python if deploy on Ubuntu.] *************************************
changed: [localhost]

PLAY [env-hosts] ***************************************************************

TASK [setup] *******************************************************************
ok: [localhost]

TASK [bootstrap : Perform cleanup tasks if cleanup=true.] **********************
skipping: [localhost]

TASK [bootstrap : Bootstrap tasks for RedHat-based systems.] *******************
skipping: [localhost]

TASK [bootstrap : Bootstrap tasks for Debian-based systems.] *******************
included: /root/work/django-project-deploy/roles/bootstrap/tasks/debian.yml for localhost

TASK [bootstrap : Install system updates.] *************************************
changed: [localhost]

TASK [bootstrap : Install required packages.] **********************************
ok: [localhost] => (item=[u'apt-transport-https', u'ca-certificates', u'curl', u'software-properties-common', u'git'])

TASK [bootstrap : Add gpg key of Docker repository.] ***************************
ok: [localhost]

TASK [bootstrap : Add Docker repository.] **************************************
ok: [localhost]

TASK [bootstrap : Install Docker-ce package.] **********************************
ok: [localhost]

TASK [bootstrap : Start and enable required services.] *************************
ok: [localhost] => (item=docker)

TASK [postgres : Create required direcories.] **********************************
changed: [localhost]

TASK [postgres : Perform init tasks.] ******************************************
skipping: [localhost]

TASK [django : Create required direcories.] ************************************
changed: [localhost]

TASK [django : Perform init tasks.] ********************************************
skipping: [localhost]

TASK [nginx : Create required direcories.] *************************************
changed: [localhost]

TASK [nginx : Create Nginx config from template.] ******************************
changed: [localhost]

TASK [nginx : Perform init tasks.] *********************************************
skipping: [localhost]

PLAY RECAP *********************************************************************
localhost                  : ok=13   changed=6    unreachable=0    failed=0

$ docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
```

**Comment by kisev** [comment]
Hi, thanks for reply.
Yes, it's my fault. I added additional task to bootstrap role to detect first deploy.
Also cleanup option described more accurately in readme file.

Be careful, this playbook mainly intended for clean separate virtual machine or host (as I expect). If you have sensitive data in /var/lib/docker, please tell me and I can choose different way for totally cleanup target host before deployment.

Also because you started playbook before cleanup detection improvements, you should pass -e cleanup=true option first.

**Comment by 1it** [comment]
Thanks, it works now.
I think cleaning up all current Docker data is a very rude decision.
Usually, developers just want to run one script and get a working project.

**Comment by kisev** [comment]
I Agree, would be better if I remove only persistent data (already in master branch).
This project just POC, and not support several things important for local development:
- Support more than one project per host (it's not too complicated but not implemented in first version).
- Docker-machine support for dynamically create local (or remote) vm's with docker for containers (possible solution for previous problem but also for more comfortable use).

**Comment by kisev** [comment]
Bug fixed by 35a25856d807930b0d873422cdf1f4c86dee49ec
